cdn/dev/js/kmwlive.js

```diff
@@ -241,6 +240,28 @@ $(document).ready(function() {
   window.setInterval(refreshButtons,200);  
 });
 
+// The KeymanWeb script will have loaded by this point, though initialization may be another matter.
+// Promises only started being supported in KMW 13 (keymanapp/keyman#1432), but this site supports
+// earlier versions, too, so we can't rely on their presence.
```

Comment:
Happy for this site to support 13.0+ at this point if that makes things more elegant?